### PR TITLE
chore: minor style updates & fix a windows test

### PR DESCRIPTION
**Pull request by marktnoonan**
This PR fixes a windows Launchpad test that was failing due to config being slow to load. It's only possible to test the "record first run" prompt after a testing type has been chosen, which means the test needs to wait for the "Choose a Browser" screen to appear.

Also tweaks the border of the Major Version Welcome page to match the color in figma, and removes an extra overflow utility class that was causing a double scrollbar on Windows.

### PR Tasks

- [x] Have tests been added/updated?
- [na] Has the original issue (or this PR, if no issue exists) been tagged with a release in ZenHub? (user-facing changes only)
- [na] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [na] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?

### CODE REVIEW

**Comment by lmiller1990** [comment]
Double scroll bar is gone. It says "11.0.0 Released next month", but users will see it this month. Do we even need a byline? How about just "11.0.0 - latest release"?

image

**Review by lmiller1990 — COMMENTED** [review]
Fix works, left comment about other wording.

**Comment by warrensplayer** [comment]
> Double scroll bar is gone. It says "11.0.0 Released next month", but users will see it this month. Do we even need a byline? How about just "11.0.0 - latest release"?
> 
> image

I fixed the values being passed to Date.UTC that was a parameter to useTimeAgo. The `monthIndex` parameter is `0` based. The display will be a little strange during the day tomorrow because it will report that the release was done at midnight UTC, but then will be correct as time progresses.

After fix:
image

**Comment by marktnoonan** [comment]
Good catch @warrensplayer! We need to update the test for "11.0.0 Released just now"

File: system-tests/projects/react17/package.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
     "@vitejs/plugin-react": "^1.3.2",
     "css-loader": "^4.3.0",
     "style-loader": "^2.0.0",
-    "vite": "2.9.14",
+    "vite": "3.2.2",
```
Comment (ZachJW34) [inline system-tests/projects/react17/package.json:10]: Why was this updated? Should bump `@vitejs/plugin-react` as well

Comment (marktnoonan) [reply system-tests/projects/react17/package.json:10]: Some context here in slack: https://cypressio.slack.com/archives/C011BAPC614/p1667810101355419?thread_ts=1667789779.018379&cid=C011BAPC614

File: packages/app/cypress/e2e/support/execute-spec.ts [hunk 2]
```diff
@@ -23,13 +22,10 @@ declare global {
   }
 }
 
-// Here we export the function with no intention to import it
-// This only tells the typescript type checker that this definitely is a module
-// This way, we are allowed to use the global namespace declaration
-export const waitForSpecToFinish = (expectedResults, timeout?: number) => {
+const waitForSpecToFinish = (expectedResults, timeout?: number) => {
   // First ensure the test is loaded
-  cy.get('.passed > .num').should('contain', '--')
```
Comment (lmiller1990) [inline packages/app/cypress/e2e/support/execute-spec.ts:31]: Race condition - sometimes the rest actually executes fast enough, and it's not `--`, but the number of passes. So, better just verify the elements exist.

File: system-tests/projects/vueclivue2-configured/package.json [hunk 3]
```diff
@@ -10,8 +10,6 @@
     "vue": "^2.6.11"
   },
   "devDependencies": {
-    "@cypress/vue2": "file:../../../npm/vue2",
```
Comment (lmiller1990) [inline system-tests/projects/vueclivue2-configured/package.json:13]: This doesn't do what you expect on windows apparently. Also, seems like it isn't strictly required.

**Review by lmiller1990 — APPROVED** [review]
I messed around a bit to get windows to a reasonable state. There is some flake I could not solve, but it's windows only and the behaviors are working fine and verified via linux tests; I skipped them and created issues in the interest of getting this merged.

- https://github.com/cypress-io/cypress/issues/24575
- https://github.com/cypress-io/cypress/issues/24579

Looks like there is still some flake, but far less than before.

I did `skip ci` for the last commit. I checked CI before merging; everything is ✅ .

**Pull request merged**